contacts_vs_distance.ipynb

```diff
@@ -44,27 +44,29 @@
    "outputs": [],
```

Comment:
<p>I think what needs to be thought through is the number of contacts shown on the figure vs which one or multiple regions are plotted... Also, I would add the deirvative curves, it looks to me like that will be noisy in the 0.01% sample.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>